src/components/HyloTipTapEditor/HyloTipTapEditor.js

```diff
@@ -0,0 +1,213 @@
+import React, { useRef, useImperativeHandle, useEffect, useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { isEmpty } from 'lodash/fp'
+import { useEditor, EditorContent, Extension, BubbleMenu } from '@tiptap/react'
+import Highlight from '@tiptap/extension-highlight'
+import Placeholder from '@tiptap/extension-placeholder'
+import StarterKit from '@tiptap/starter-kit'
+import { VscPreview } from 'react-icons/vsc'
```

Comment:
Using `react-icons` is a break from our "icomoon" pattern, but I really like it. Super helpful for building a menubar for the editor, but it has some general advantages which we should discuss at some point.